src/hooks/usePagination.example.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useEffect } from 'react';
 import { usePagination } from './usePagination';
 
-export default function UsePaginationExample({ totalPages = 1 }) {
-  const {
+export default function UsePaginationExample() {
+  let {
```

Comment:
why `let`?

Reply:
To allow for default values. That assignment inside the destructure didn't work, so I had to add something lower down to reassign it to 1 if it was undefined.

Reply:
ohh interesting